services/datamanager/datacapture/data_capture_file.go

```diff
@@ -33,8 +32,8 @@ const (
 	nextPointCloud = "NextPointCloud"
 	pointCloudMap  = "PointCloudMap"
 	// Non-exhaustive list of characters to strip from file paths, since not allowed
-	// on at least Windows and Darwin.
-	windowsDarwinReservedChars = ":"
+	// on at least Windows, Darwin, and Ubuntu 20.04.
```

Comment:
I would say "on certain file systems". From what I understand (albeit from brief Googling), the filesystem is what limits this, not the OS. I found [this explanation ](https://superuser.com/questions/292748/is-file-system-part-of-operating-system) helpful:

> The filesystem itself, represented by the physical order of information on a storage-representation, is independent of the OS. The OS contains a driver that allows it to work with the filesystem. Some filesystems may only have one OS that can talk to it, and that OS has that filesystem hardcoded into it (think Novell NetWare's original filesystem); but that doesn't stop some enterprising person from writing such a driver for another OS just because.

So I think any OS that tries to write to the USB will have this issue regardless of whether the operating system typically is shipped with a filesystem that can support writing files with colons in their names. I'm on MacOS/Darwin and am actually able to write files with colons on my local file system, just not on the USB.

Reply:
Thanks for digging into this! Updated in the export CLI as well, and tested. Way cleaner.